dev/release/rat_exclude_files.txt

```diff
@@ -116,6 +116,7 @@ ci/*
 **/*.svg
 **/*.csv
 **/*.json
+**/*.sql
```

Comment:
I _think_ this is the correct thing to do, but someone should probably verify if RAT is needed for SQL files

Reply:
This is fine in my opinion